contents/site/meta/wiki_sunset.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "diaspora* Wiki Sunset"
+subtitle: The information you requested got relocated.
+---
+
+<%=
+  partial "components/inline_image",
+  alt: "We are vewwy sowwy.",
+  src: static_url("site/mr_abdoun_the_scottish_cat.jpg")
+%>
+
+In Spring 2019, we decided to discontinue the diaspora\* Project Wiki and instead merge the information with the project site and the dedicated installation guides. This allows us easier control and maintenance of the project's infrastructure, as well as streamlining contributions to both the wiki and our project sites. While we attempted to keep the old URLs working by setting up redirects, we could not redirect all wiki links. If you are reading this, it means you followed a link to an old wiki resource that we could not reallocate directly. Sorry about that.
```

Comment:
A number of changes:

In Spring 2019, we decided to discontinue the diaspora\* project wiki and to merge the information it contained with the project site and our dedicated installation guides. This makes maintenance of the project's infrastructure far easier, as well as streamlining content. While we attempted to keep old URLs working by setting up redirects, we could not redirect all wiki articles to pages on our new site. If you are reading this, it means you followed a link to an old wiki resource that we could not reallocate directly. Sorry about that.